card/mod/carrierwave/set/type/image.rb

```diff
@@ -119,16 +121,16 @@ def new_image action
 format :file do
   include File::FileFormat
 
-  def image_style
-    ["", "full"].member?(params[:size].to_s) ? :original : params[:size].to_sym
-  end
-
-  def selected_file_version
-    style = voo.size = image_style.to_sym
-    if style && style != :original
-      card.attachment.versions[style]
-    else
-      card.attachment
-    end
-  end
+  #def image_style
+  #  ["", "full"].member?(params[:size].to_s) ? :original : params[:size].to_sym
```

Comment:
We drop support for `full` and a size parameter?

Reply:
Still supported!  The old (current) implementation has both `selected_version` and `selected_file_version` doing the same thing.  See file.rb, where the `selected_file_version` is replaced with `selected_version` (which file and image handle differently)